applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -26,6 +26,132 @@ config BLE_LE_POWER_CONTROL_ENABLED
 	 dynamically and automatically during connection, which provides effective
 	 communication.
 
+choice	BLE_CONN_TX_POWER
+	prompt "Default tx power for BLE connections"
+	default BLE_CONN_TX_POWER_0DBM
+	help
+		Set the default tx power for BLE connections.
+
+config BLE_CONN_TX_POWER_0DBM
+	bool "0dBm"
+
+config BLE_CONN_TX_POWER_NEG_1DBM
+	bool "-1dBm"
+
+config BLE_CONN_TX_POWER_NEG_2DBM
+	bool "-2dBm"
+
+config BLE_CONN_TX_POWER_NEG_3DBM
+	bool "-3dBm"
+
+config BLE_CONN_TX_POWER_NEG_4DBM
+	bool "-4dBm"
+
+config BLE_CONN_TX_POWER_NEG_5DBM
+	bool "-5dBm"
+
+config BLE_CONN_TX_POWER_NEG_6DBM
+	bool "-6dBm"
+
+config BLE_CONN_TX_POWER_NEG_7DBM
+	bool "-7dBm"
+
+config BLE_CONN_TX_POWER_NEG_8DBM
+	bool "-8dBm"
+
+config BLE_CONN_TX_POWER_NEG_12DBM
+	bool "-12dBm"
+
+config BLE_CONN_TX_POWER_NEG_16DBM
+	bool "-14dBm"
+
+config BLE_CONN_TX_POWER_NEG_20DBM
+	bool "-20dBm"
+
+config BLE_CONN_TX_POWER_NEG_40DBM
+	bool "-40dBm"
+
+endchoice
+
+config BLE_CONN_TX_POWER_DBM
```

Comment:
Why do we need both BLE_CONN_TX_POWER and BLE_CONN_TX_POWER_DBM? Why cant we just use BLE_CONN_TX_POWER_DBM?
Same for BLE_ADV_TX_POWER/BLE_ADV_TX_POWER_DBM

Reply:
One is used as a choice, the one that users using menuconfig/guiconfig will see. and the other is the actual integer value used in the code. The reason for using a choice is to restrict the user to the power settings we actually have, instead of allowing them to set whatever value they want and then floor it to the nearest valid value